pickhardtpayments/OracleLightningNetwork.py

```diff
@@ -71,3 +71,22 @@ def theoretical_maximum_payable_amount(self, source: str, destination: str, base
 
         mincut, _ = nx.minimum_cut(test_network, source, destination)
         return mincut
+
+    def settle_payment(self, path: OracleChannel, payment_amount: int):
```

Comment:
so a `path` is a `List` of `OracleChannels`? otherweise line 84 `for channel in path` will not work

Reply:
yes, path is a <class 'list'>